src/odemis/gui/cont/export.py

```diff
@@ -145,13 +147,19 @@ def _get_export_info(self):
             else:
                 # TODO: remove numbers from the view name?
                 basename += " " + view.name.value
+
+            # only batch export when more than 1 image to export for view
+            if hasattr(streams[0], "polarimetry") or \
```

Comment:
Could you put parenthesis? It's not clear whether it does "(A or B) and C",  or "A or (B and C)". That's quite a bit different.
Also, if that's the former, then something is weird because the stream might not have a .polarization, and still the lenght of polarization.choices is checked.

Reply:
No it is the second one.